### DISCUSSION TITLE
Noobs - Look Here

### DISCUSSION THREAD

**Discussion by 21Machines** [bmaltais/kohya_ss#2832] [Q&A]
I'm going to save you a lot of headaches.  Once you've installed Kohya_ss and you are staring at the GUI.  If you by chance, you did not select the Lora Tab at the top AND you loaded your config file into the DreamBooth tab by mistake.  You MUST kill your instance in the terminal and restart.  There is no coming back from this loading mistake.  Also learn the config.toml file in the root, modify your paths using this file and use the same paths every time.  

Folders structure:  My main folder is called "training_directory", in that folder in have three subfolders listed below.  The fourth  "10_TOK woman" folder is nested in the img folder.   

> training_directory
>  > img >  10_TOK woman (This folder I have my prepared images and .txt files)  name your folder for "repeats_trigger_concept"
>  > log
>  > model

IMPORTANT:  When asked to point to your images, point to the subfolder 'img" not "10_TOK woman".

IMPORTANT:  If you get CUDA_OUT_OF_MEMORY warning and then an Exit.  Restart your instance at the terminal.

**Comment by kalle07** [comment]
image folder is old stuff if you have whatched once a kohya youtube video
